Xwt.Gtk.Mac/Xwt.Gtk.Mac.csproj

```diff
@@ -78,5 +79,9 @@
       <Project>{92494904-35FA-4DC9-BDE9-3A3E87AC49D3}</Project>
       <Name>Xwt</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Xwt.XamMac\Xwt.XamMac.csproj">
```

Comment:
This is not right, `Xwt.Gtk.Mac` should not depend on `Xwt.XamMac`, also problematic with packages.